201-api-management-logs-to-moesif-using-eventhub-webapp/nested/microsoft.web/sites/extensions.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appServiceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Azure App service resource"
+            }
+        },
+        "webJobZipDeployUrl": {
+            "type": "string",
+            "metadata": {
+                "description": "URL to Zip file that contains webjob using format suitable to be published using msdeploy"
+            }
+        },
+        "tags": {
+            "type": "Object",
+            "defaultValue": {}
+        }
+    },
+    "functions": [],
+    "variables": {},
+    "resources": [
+        { // This deployment sometimes reports deployment failure, even though it may have succeeded (perhaps eventually)
```

Comment:
why?

Reply:
Please see my response to your question in README.. this is the MSDeploy issue.